### ISSUE TITLE
anti-apoCII autoantibody

### ISSUE THREAD

**Issue by MaudDLVT** [obophenotype/human-phenotype-ontology#10512]
**Preferred term label:**
anti-apolipoprotein CII autoantibody positivity

**Synonyms**
anti-apoCII autoantibody positivity

**Definition (free text, please give PubMed ID)**
The presence of autoantibodies in the serum that react against apoCII
PMID: 30706594

**Parent term (use hpo.jax.org/app)**
HP:0030057

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:444490

**Your nano-attribution (ORCID)**
0000-0002-3387-1836

**Comment by pnrobinson** [comment]
@MaudDLVT 
Adding new term: 
- ID: HP:6001061
- label: Anti-apolipoprotein CII autoantibody positivity
- parents: [Autoimmune antibody positivity (HP:0030057)](https://hpo.jax.org/app/browse/term/HP:0030057)
- pmid: PMID:30706594
- def: The presence of autoantibodies in the blood circulation that react against apoCII.
- comment: 
- orcid: orcid:0000-0002-3387-1836
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10512
- synonyms: Anti-apoCII autoantibody positivity(NONE)
